repository/redis/0.1.0/operator.yaml

```diff
@@ -0,0 +1,30 @@
+name: redis
+version: 0.1.0
+kudoVersion: 0.3.0
+maintainers:
```

Comment:
@djannot was originally a maintainer/1stcommitter so would love to see him as well in this list :)

Reply:
Thanks for adding me, I'll take a look